client/client.go

```diff
@@ -369,6 +372,17 @@ const (
 	WorkflowIDReusePolicyRejectDuplicate WorkflowIDReusePolicy = internal.WorkflowIDReusePolicyRejectDuplicate
 )
 
+const (
+	// Default subject to change in the server config
+	ParentClosePolicyDefault ParentClosePolicy = internal.ParentClosePolicyDefault
```

Comment:
Remove ParentClosePolicyDefault and default the value to ParentClosePolicyTerminate

Reply:
For backward compatibility, do you want to let server always override terminate to be abandon when seeing a policy is terminate no matter what? There could be cases that customers specify terminiate as policy and we won't be able to distinguish them if we make the default to terminate.